src/frontend/src/pages/CalendarPage/DesignReviewCreateModal.tsx

```diff
@@ -127,16 +127,21 @@ export const DesignReviewCreateModal: React.FC<DesignReviewCreateModalProps> = (
 
   const memberOptions = users.map(userToAutocompleteOption);
 
+  const wbsDropdownOptionsSorted: { label: string; id: string }[] = [];
   const wbsDropdownOptions: { label: string; id: string }[] = [];
 
   allWorkPackages.forEach((workPackage: WorkPackage) => {
-    wbsDropdownOptions.push({
+    wbsDropdownOptionsSorted.push({
       label: `${wbsNamePipe(workPackage)}`,
       id: wbsPipe(workPackage.wbsNum)
     });
   });
 
-  wbsDropdownOptions.sort((wp1, wp2) => wbsNumComparator(wp1.id, wp2.id));
+  wbsDropdownOptionsSorted.sort((wp1, wp2) => wbsNumComparator(wp1.id, wp2.id));
```

Comment:
change the sorting comparator so instead of .sort((wp1,wp2) do .sort(wp2, wp1) this will reverse the order